### PR TITLE
Feat/e2e test mobile

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/views/Me/DraftDetail/index.tsx [hunk 1]
```diff
@@ -148,7 +148,7 @@ const DraftDetail = () => {
 
   return (
     <Layout.Main
-      aside={<Sidebar draft={draft} ownCircles={ownCircles} />}
+      aside={isLargeUp && <Sidebar draft={draft} ownCircles={ownCircles} />}
```
Comment (robertu7) [inline src/views/Me/DraftDetail/index.tsx:151]: `display: none` on `aside` can be located by Playwright?

File: tests/helpers/poms/articleDetail.ts [hunk 2]
```diff
@@ -77,6 +77,9 @@ export class ArticleDetailPage {
     this.toolbarSupportButton = this.page.getByRole('button', {
       name: 'support author',
     })
+    // this.toolbarCommentButton = this.page.getByTestId(
```
Comment (robertu7) [inline tests/helpers/poms/articleDetail.ts:80]: Not implemented yet?

File: tests/publishDraft.spec.ts [hunk 3]
```diff
@@ -10,98 +10,101 @@ import {
...
 
-    // Goto published article page
-    // Promise.all prevents a race condition between clicking and waiting.
-    await Promise.all([
-      draftDetail.dialogViewArticleButton.click(),
-      page.waitForNavigation(),
-    ])
+      // Goto published article page
+      // Promise.all prevents a race condition between clicking and waiting.
+      await Promise.all([
```
Comment (robertu7) [inline tests/publishDraft.spec.ts:65]: looks like we wrong the order of `Promise.all` here, we should place network waiting before clicking:

```ts
await Promise.all([
  page.waitForNavigation(),
  draftDetail.dialogViewArticleButton.click(),
])
```

Comment (robertu7) [reply tests/publishDraft.spec.ts:65]: same issue as https://github.com/thematters/matters-web/pull/3102/files#diff-e38da8c9744ea0902ea1dcabf4ebf25e0ba33cca68121172bd009ab3b59190f5R259-R265

**Review by robertu7 — APPROVED** [review]

File: src/views/Me/DraftDetail/index.tsx (hunk 1 above)
Comment (pitb2022) [reply src/views/Me/DraftDetail/index.tsx:151]: Yes, it can be located by Playwright.

File: tests/helpers/poms/articleDetail.ts (hunk 2 above)
Comment (pitb2022) [reply tests/helpers/poms/articleDetail.ts:80]: deprecated

File: tests/publishDraft.spec.ts (hunk 3 above)
Comment (pitb2022) [reply tests/publishDraft.spec.ts:65]: done

**Pull request merged**